features/visual-search.md

```diff
@@ -0,0 +1,69 @@
+# Visual search
+
+{% hint style="info" %}
+**Note**
+
+This feature is only available on request for specific plans.\
+For more information, please contact the [support team](mailto:support@imagekit.io) or your ImageKit Customer Success Manager.
+{% endhint %}
+
+Visual search allows you to search for similar images based on visual content. Instead of relying on text-based keywords or metadata, you can search based on visual elements like colors, shapes, textures, and any other information derived from the image.
+
+By searching visually rather than by name or metadata, you can:
+
+* Easily locate images without needing to remember the tags or names of the files.
+* Search for similar images in your media library.
+* Identify duplicates in your media library with ease.
+* Discover images in your media library through visual characteristics.
+* Group visually similar images together for better organization.
+* Enhance the discoverability of your assets.
+
+You can perform visual search from the dashboard either by text or by image:
+
+*   **Searching by Text**: You can type in a word or a phrase to find images that visually match it.\
+    For example, you can type in `pink birds`; all the images related to `pink birds` will be retrieved from your media library.\
+
+
+    <figure><img src="../.gitbook/assets/text_search.png" alt=""><figcaption></figcaption></figure>
+
+    <figure><img src="../.gitbook/assets/visual-search/text_search_results.png" alt=""><figcaption></figcaption></figure>
```

Comment:
This image result isn't very good. Can we use some e-commerce or design example?

Reply:
One file element is also distorted on the bottom right.

Reply:
Also cropping of screens is a bit inconsistent. Some has the header toolbar and some don't.

Moreover, I think we should put markers on the image search icon. And even in text search we can show that AI search option appears once you start typing.